### DISCUSSION TITLE
Problem resizing the remote connection window.

### DISCUSSION THREAD

**Discussion by Jedmmota** [rustdesk/rustdesk#9061] [General]
I was unable to create a new issue, so I am posting it here.

Recently in the Windows 11 version (1.3.0 nightly) I noticed a change in RustDesk's behavior when I want to resize the remote connection window. Before, resizing was easy, as it should be in any window. Now it has become extremely difficult to resize, as can be seen in the attached video.

To reproduce: Try resizing the remote connection window.
Expected Behavior: Difficulty resizing the window. The cursor does not behave as expected.

https://github.com/user-attachments/assets/d6bc19dd-08f5-4cb8-8465-5d38ddaf5076

**Comment by rustdesk** [comment]
@fufesou really? The only possible is still that PR.

**Comment by Jedmmota** [comment]
I only noticed this yesterday, but it may have started happening before.

**Reply by rustdesk** [reply]
Please show your `settings` -> `About` page (local side)

**Reply by Jedmmota** [reply]
![image](https://github.com/user-attachments/assets/5d6463c4-573a-4307-bfad-8421f0442243)
This build was generated by GitHub itself.

**Comment by fufesou** [comment]
https://github.com/rustdesk-org/rustdesk_desktop_multi_window/pull/22

Sorry,  I removed it because I was confused about the code when fixing the bug.

**Reply by Jedmmota** [reply]
I just implemented the changes you made and compiled again. Window resizing works perfectly again. Thank you very much @rustdesk and @fufesou.
